baton/handler.go

```diff
@@ -27,5 +30,7 @@ func serveWs(w http.ResponseWriter, r *http.Request) {
 	c := conn{send: make(chan []byte, 256), ws: ws}
 	h.register <- c
 	go c.writePump()
+	c.send <- []byte(strconv.Itoa(i))
```

Comment:
Okay, before we deploy this I think I'd be most comfortable if all data sent over the wire was JSON.